.github/workflows/build.yml

```diff
@@ -35,10 +57,8 @@ jobs:
       - name: Build Legacy
         run: GOARCH=${{ matrix.go-arch }} COSMOS_BUILD_OPTIONS=legacy make build
       - name: Build with rocksdb backend
-        if: |
-          env.GIT_DIFF &&
```

Comment:
Not sure about this condition. seems to be copy pasted from linting workflow.

Reply:
yep, that was invalid 👍🏾